srcs/utils/exit_error.c

```diff
@@ -43,9 +43,16 @@ static void	st_write_msg(const char *fn, const char *msg)
  * @param function	(optional) Function where the error was encountered.
  * @param message	(optional) Custom error message to display.
 */
-int	exit_error(const int code, const char *function, const char *message)
+int	logger(const int code, const char *function, const char *message)
 {
 	if (function || message)
 		st_write_msg(function, message);
 	return (code);
 }
+
+int	exit_error(const char *message)
+{
+	if (message)
+		st_write_msg(NULL, message);
```

Comment:
Logger() was supposed to replace st_write_msg right?

Reply:
So there's logger and exit error, which both use st_write_error to output either a message either a function name and/or a message

Reply:
But we might not need both as this was used for my minishell to get an appropriate error code and message

Reply:
yeah this is fine, but writing the msg is should be logger, and the exit_error is for exit code, maybe exit_code_error()